src/screens/SendSheet.js

```diff
@@ -580,11 +580,16 @@ export default function SendSheet(props) {
           Alert.alert(lang.t('wallet.transaction.alert.invalid_transaction'));
           submitSuccess = false;
         } else {
-          const { result: txResult } = await sendTransaction({
+          const { result: txResult, error } = await sendTransaction({
             existingWallet: wallet,
             provider: currentProvider,
             transaction: signableTransaction,
           });
+
+          if (error) {
+            throw new Error(`SendSheet sendTransaction failed`);
+          }
+
           const { hash, nonce } = txResult;
```

Comment:
Reading `{ hash }` from undefined `txResult` was throwing prior to this, which is a bit of a red herring. More clear to throw an explicit error.